### ISSUE TITLE
Addon-a11y - Disable accessibility tab in a single story

### ISSUE THREAD

**Issue by Aekal** [storybookjs/storybook#6849]
**Describe the bug**
I can't find a way to disable accessibility tab in a single story. 
In `config.js` I have defined decorator `addDecorator(withA11y)` and in some stories I want to disable this tab. For example I have a simple component like input without label, so I don't need to see violations related to missing label.
I tried to use `disable` option as in info addon (tab is empty there), but it doesn't seem to work:
```
.addParameters({
  a11y: {
    disable: true
  }
})
.addParameters({
  info: {
    disable: true
  }
})
```
and this code disables all rules not only in single story but in every one
```
.addParameters({
  a11y: {
    config: { // axe-core configurationOptions (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1)
      disableOtherRules: true
    }
  }
})
```
And going further I have a second question. How to pass specific axe rules for only a single story?

**Expected behavior**
Hide or leave empty accessibility tab when `disable: true` option is passed

**System:**
 - OS: MacOS
 - Framework: Vue
 - Addons: A11y
 - Version: 5.0.11

**Comment by shilman** [comment]
Whoopee!! I just released https://github.com/storybookjs/storybook/releases/tag/v5.2.0-alpha.36 containing PR #6923 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Ermahgerd!! I just released https://github.com/storybookjs/storybook/releases/tag/v5.2.0-alpha.37 containing PR #7333 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.
